### ISSUE TITLE
Indication when text is copied to the clipboard

### ISSUE THREAD

**Issue by stevenbarragan** [warpdotdev/warp#152]
### Is your feature request related to a problem? Please describe.

```shell
I'd like to have some visual feedback when text gets copied to the clipboard
```

### Describe the solution you'd like?

```shell
Display a sort of alert when text is copied into the clipboard
```

### Additional context

It will make users more aware of it when it happens

**Comment by zheng** [comment]
Yes, I think we can definitely show a toast similar to "Link Copied!". Thanks for filing

**Comment by dannyneira** [comment]
Please :+1:  and subscribe to this request as it helps us gauge interest. We will post any updates here.
